hbase/Dockerfile

```diff
@@ -208,7 +193,10 @@ USER ${STACKABLE_USER_UID}
 WORKDIR /stackable
 
 COPY --from=hadoop-builder --chown=${STACKABLE_USER_UID}:0 \
-  /stackable/hadoop/share/hadoop/tools/lib/aws-java-sdk-bundle-*.jar \
+  # The artifact name of the AWS bundle has changed between Haddop 3.3.6 and 3.4.1
+  # from aws-java-sdk-bundle-*.jar to bundle-*.jar.
+  # See: https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/aws_sdk_upgrade.html
+  /stackable/hadoop/share/hadoop/tools/lib/bundle-*.jar \
```

Comment:
Just double checking this, since we've decided to stick to the same Hadoop version.

Reply:
yeah, sorry for the omission https://github.com/stackabletech/docker-images/pull/1101/commits/3156f96dde789fc19803d9481992b614a9a05ba0